src/include/test/unit_ivf_qv.cc

```diff
@@ -302,6 +298,143 @@ TEST_CASE("ivf qv: finite all or none", "[ivf qv][ci-skip]") {
       CHECK(std::labs(intersections00 - intersections04) < 12);
       CHECK(std::equal(D00.data(), D00.data() + D00.size(), D04.data()));
     }
+
+    SECTION(
+        "detail::ivf::qv_query_heap_finite_ram, counting_l2_distance "
```

Comment:
Can we also add tests for `inner_product_distance` here?

Reply:
See comment above.